backend/btrixcloud/emailsender.py

```diff
@@ -35,11 +37,13 @@ def _send_encrypted(self, receiver, subject, message):
         msg.set_content(message)
 
         context = ssl.create_default_context()
-        with smtplib.SMTP(self.smtp_server, 587) as server:
+        with smtplib.SMTP(self.smtp_server, self.smtp_port) as server:
+            if self.smtp_use_tls:
+                server.ehlo()  # Can be omitted
```

Comment:
Seems like it's good practice so I'd keep the command and drop the comment

Reply:
Sure, yeah, i guess can just drop the comment since we kept it in there thus far.